application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -2645,6 +2657,8 @@ type step {
   distance: Float
   "The elevation profile as a list of { distance, elevation } values."
   elevationProfile: [elevationProfileComponent]
+  "Information about an station entrance or exit"
+  entrance: Entrance
```

Comment:
Unless I'm totally mistaken, I think on this API level we still want to have the entity field which returns an entity union.